CMakeLists.txt

```diff
@@ -15,7 +15,7 @@
 cmake_minimum_required(VERSION 3.5)
 
 project(static_exception)
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
@kylemarcey I've bumped this because from what I can tell `aligned_alloc` is only available in C++17 I think.